docs/user_guides/benchmarking/docker.rst

```diff
@@ -0,0 +1,59 @@
+.. *benchmarking*\ docker:
```

Comment:
I suspect this reference is not right. Should it not be `.. _benchmarking_docker:`?